docs/migrateNode10.md

```diff
@@ -111,4 +111,9 @@ The following `fs` functions all have incompatibilities. In addition, any other
 - fs.unlink
 - fs.utimes
 - fs.write
-- fs.writeFile
\ No newline at end of file
+- fs.writeFile
+
+# How to disable warnings for Node 6 deprecation
```

Comment:
Could you please update the table of content as well?

Reply:
Updated, thanks!